.travis.yml

```diff
@@ -45,6 +45,7 @@ install:
   - ./scripts/launch-contentpool.sh
   - ./scripts/run-server.sh
   - ./scripts/init-project.sh
+  - cd ../contentpool-project && source dotenv/loader.sh && docker-compose exec web drush upwd dru_admin changeme && cd -
```

Comment:
Is this really necessary to add? If so, it should go into the suiting script above.

Reply:
If you know better way how to obtain password or login, then please let me know. The problem with this test is that it needs access to both contentpool and satellite, but you have real behat session only to one of them. So I cannot use "I am logged in as a user with the "administrator" role" for both sites, only for one of them, and for the second I just need to manually log in. But don't have password, because it's only printed once after site install. That's why I just change it. Which one is that suiting script then? I guess none of them, because they shouldn't change the password?

Reply:
Problem is this is yet another step that needs to be done, so everything works locally. Let's rather just set the password to "changeme" in the contentpool's init-project.sh script by default. We already have that for the replicator afaik and the script is for development purposes only anyway.